jibri/rootfs/defaults/config.json

```diff
@@ -26,7 +26,7 @@
             "control_muc": {
                 "domain": "{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}",
                 "room_name": "{{ .Env.JIBRI_BREWERY_MUC }}",
-		// MUST be unic for every instanse
+                // MUST be unic for every instanse
```

Comment:
unnecessary change

Reply:
Yeah you are right. But I will keep it because the formatting in that file will be be clean then ;)